sqlmesh/dbt/loader.py

```diff
@@ -105,7 +111,31 @@ def _load_models(
         return models
 
     def _load_audits(self) -> UniqueKeyDict[str, Audit]:
-        return UniqueKeyDict("audits")
+        audits: UniqueKeyDict = UniqueKeyDict("audits")
+
+        project = self._load_project()
+        context = project.context
+
+        logger.info("Converting audits to sqlmesh")
+        for package in project.packages.values():
+            context.variables = package.variables
+            for test in package.tests.values():
+                logger.info(f"Converting {test.name} to sqlmesh format")
```

Comment:
[Nit] Add quotes for `{test.name}`. Also use `%s` format for logging instead of string interpolation.